scripts/protect-branches.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+
+import argparse
+import json
+import os
+import re
+import requests
+import sys
+import urllib
+
+
+# API token received from github. For the "scopes", I only selected the entire "repo" section.
+# https://github.com/settings/tokens
+TOKEN=""
```

Comment:
I thought about this too. It's a little inconvenient to set an environment variable without putting the value in your shell history, but it's doable. I could use ``getpass()`` perhaps which would be secure, but would make you enter it every time. Or both.

Part of my dilemma is that I wrote this little script to save time and thus tried to avoid feature creep, but I agree this is worth considering.